{"name": "Создание пользователя который уже есть в системе", "status": "passed", "description": "При создании дублирующего пользователя срабатывает allert ", "start": 1712571498230, "stop": 1712571499067, "uuid": "4956be54-f9f0-450e-9217-e18453a787ea", "historyId": "511af5c3d1ba0594e424965a597653ae", "testCaseId": "511af5c3d1ba0594e424965a597653ae", "fullName": "tests.test_create_user.TestCreateUser#test_create_double_user_error", "labels": [{"name": "suite", "value": "Создание пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "Mi-Notebook"}, {"name": "thread", "value": "1580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}